Makefile

```diff
@@ -1,12 +1,22 @@
 all: test
 
-uninstall:
-	pip freeze | grep -v "^-e" | xargs pip uninstall -y
-
-install:
+install.tools:
 	pip install --upgrade pip twine wheel
+
+## Install openfisca-core for deployment and publishing.
+install.build: setup.py clean install.tools
+	@## This allows us to be sure tests are run against the packaged version
+	@## of openfisca-core, the same we put in the hands of users and reusers.
+	pip install --upgrade pip wheel
+	python $< bdist_wheel
+	find dist -name "*.whl" -exec pip install --force-reinstall {}[dev] \;
+
+install: install.tools
 	pip install --editable .[dev] --upgrade --use-deprecated=legacy-resolver
 
+uninstall:
+	pip freeze | grep -v "^-e" | sed "s/@.*//" | xargs pip uninstall -y
```

Comment:
Why is this changed when it is not used anywhere? 🤔

Reply:
I use it a lot: for example to try this changeset out, you have to repeatedly do a clean slate so as to avoid false-positives. This is one of the problems that a tool like `nox` adresses quite gracefully.

It could be great if you added a bit of doc for contributors on how to "troubleshoot".

Reply:
Sorry, I wasn't clear: what I'm wondering about is why there was a change in this line, with the added `sed`, I was not questioning the usefulness of this target 🙂

Reply:
Ah, yes, I added a documentation to the task (it is to also uninstall editable versions of other packages, otherwise the task fails).